prop_angular_velocity_msg[] props
